### DISCUSSION TITLE
AI人工智能雖然很聰明，但在團隊合作協調方面仍有些問題

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS3073-C/Python-Community-GS3073-C#335] [General]
在國際象棋或圍棋等遊戲方面，人工智慧 (AI) 已遠遠超過了不少頂尖棋手。這些超人般的人工智慧無與倫比的競爭對手，但與人類合作或許比與人類競爭更難，同樣的技術能和人和諧相處嗎？

有不少研究顯示，人類在玩需要團隊合作的遊戲時，發現人工智慧可能淪為豬隊友，並對“團隊智能”構成了一大挑戰。其中在一項新的研究中，麻省理工學院林肯實驗室的研究人員試圖找出人類在使用先進的人工智慧模型玩需要扣過合作來的紙牌遊戲 Hanabi 時的表現，該模型經過訓練以強化與從陌生隊友一起玩。

在單盲實驗中，參與者玩了兩個系列的遊戲：一個是 AI 代理作為他們的隊友，另一個則是當rule-based agent且手動打code以預先定義遊戲策略的機器人。

最終結果令研究人員感到驚訝，人工智慧隊友的得分竟然比rule-based agent得來得差，而且合作過的人們大多反應討厭他們的人工智慧隊友，他們也認為人工智慧隊友是不可預測的、不可靠的且不可信的，即使團隊有得分，他們仍會感到消極。

![Hanabi_Experiment](https://user-images.githubusercontent.com/91121500/140015237-2481b8d4-26b0-49dc-8a26-6338c7f1876b.png)

資料來源:
https://news.mit.edu/2021/does-artificial-intelligence-play-well-others-1004

**Comment by EricAgumonJou** [comment]
未來若能好好掌握 AI 與人類之間的 Hanabi 之類的遊戲，那麼智慧團隊合作之開闢會有無限的可能。

**Comment by EricAgumonJou** [comment]
當然也是有成功的案例:
https://www.science.org/content/article/hone-its-collaborative-skills-ai-taking-worlds-top-video-game-players

**Comment by lx3211** [comment]
畢竟團隊合作也不是每個人類都能做的很好哈哈
團隊合作真的是門大學問

**Comment by NcuJerryKao** [comment]
AI的思維 與人們差很多
一起合作 效果不好 也蠻不易外的

不過在這裡開個玩笑 我認識一位朋友 蠻像AI的喔

**Comment by BingRong1217** [comment]
AI確實很難擁有和人類一樣的思維

**Comment by limitx0** [comment]
畢竟很難理解AI在想什麼

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by larry40305** [comment]
問題的關鍵究竟為何？所謂團隊合作協調，關鍵是團隊合作協調需要如何解讀。

**Comment by 710388** [comment]
再也不敢小瞧人機對戰了 電腦都變得比人類厲害了  有時候掛機讓電腦接手反而能逆轉 我就爛
